Sources/SwiftBuild/SWBConfiguredTargetInfo.swift

```diff
@@ -11,6 +11,32 @@
 //===----------------------------------------------------------------------===//
 
 import SWBProtocol
+import SWBUtil
+
+public struct SWBArtifactInfo {
```

Comment:
Where is the Sendable conformance? With the lifting of unsafeFlags limitations in 6.2 I wonder if we can turn on the setting to make explicit Sendable annotations a requirement now, for public types in the SwiftBuild module.